plasmapy/formulary/frequencies.py

```diff
@@ -308,32 +322,21 @@ def plasma_frequency(n: u.m**-3, particle: ParticleLike, z_mean=None) -> u.rad /
     this function and can be used as follows.
 
     >>> from plasmapy.particles import Particle
-    >>> mass = Particle("p").mass.value
-    >>> plasma_frequency.lite(n=1e19, mass=mass, z_mean=1)
+    >>> mass = Particle("p+").mass.value
+    >>> plasma_frequency.lite(n=1e19, mass=mass, Z=1)
     416329...
-    >>> plasma_frequency.lite(n=1e19, mass=mass, z_mean=1, to_hz=True)
+    >>> plasma_frequency.lite(n=1e19, mass=mass, Z=1, to_hz=True)
     662608...
     """
-
-    try:
-        m = particles.particle_mass(particle).value
```

Comment:
This line had an error in it.  As written, this line took the mass from `particle`, without any change in mass due to different numbers of electrons as given by `z_mean`.

For example, if we represented a proton with `particle="H-1"` and `z_mean=1`, then `m` would be the mass of a neutral hydrogen atom.  If we did `particle="p+"` or `particle="H-1 1+"`, then `m` would be the mass of a proton.

Using `@particle_input` corrects this.